tests/cmds/test_users.py

```diff
@@ -0,0 +1,88 @@
+import json
+
+import pytest
+from pandas.core.frame import DataFrame
+from py42.response import Py42Response
+from requests import Response
+
+from code42cli.cmds.users import _get_role_id
+from code42cli.cmds.users import _get_users_dataframe
+from code42cli.main import cli
+
+TEST_ROLE_RETURN_DATA = {"data": [{"roleName": "Customer Cloud Admin", "roleId": "12"}]}
+
+TEST_USERS_RESPONSE = {
+    "users": [
+        {
+            "userId": "1234",
```

Comment:
`userId` is a number and not a string on real responses I thought

Reply:
Yes, it is. Updated.